bugsnag-android-performance/src/main/kotlin/com/bugsnag/android/performance/context/GlobalSpanContextStorage.kt

```diff
@@ -25,6 +25,9 @@ import java.util.concurrent.atomic.AtomicReference
  *     }
  * ```
  *
+ * __WARNING__: This class is inherently unsafe with coroutines and will likely lead to misleading
```

Comment:
For users with a mix of coroutines and worker threads, is the behaviour fine if `bugsnag-android-performance-coroutines` is used alongside this context storage?

Reply:
But what would be the recommendation for apps with both? Use hybrid + the coroutine package or do they conflict?